### ISSUE TITLE
[q-implant] Integration op level tests for q-implant

### ISSUE THREAD

**Issue by Judgement9882** [ONEforALL-S003/ONE#18]
ONE 레포에 작성한 integration op level tests에 대한 댓글을 작성하기 전에 정리하는 issue 입니다.

[comment]

- 모델 출처

#### Inception : [Inception_V3](https://www.tensorflow.org/lite/guide/hosted_models?hl=ko)

# 사용하는 operator

우리는 MobileNet, ResNet, Inception 모델을 circle로 변환시켜 circledump를 통해 내부 정보를 확인했습니다.

## MobileNet
```
Operator Codes: [order] OpCodeName (OpCode Enum)
[0] AVERAGE_POOL_2D (code: 1, dep_code: 1, version: 1)
[1] CONV_2D (code: 3, dep_code: 3, version: 1)
[2] DEPTHWISE_CONV_2D (code: 4, dep_code: 4, version: 1)
[3] SOFTMAX (code: 25, dep_code: 25, version: 1)
[4] RESHAPE (code: 22, dep_code: 22, version: 1)
```

## ResNet

```
Operator Codes: [order] OpCodeName (OpCode Enum)
[0] ADD (code: 0, dep_code: 0, version: 1)
[1] CONV_2D (code: 3, dep_code: 3, version: 1)
[2] MAX_POOL_2D (code: 17, dep_code: 17, version: 1)
[3] MUL (code: 18, dep_code: 18, version: 1)
[4] PAD (code: 34, dep_code: 34, version: 1)
[5] MEAN (code: 40, dep_code: 40, version: 1)
[6] RESHAPE (code: 22, dep_code: 22, version: 1)
```

## Inception

```
Operator Codes: [order] OpCodeName (OpCode Enum)
[0] AVERAGE_POOL_2D (code: 1, dep_code: 1, version: 1)
[1] CONV_2D (code: 3, dep_code: 3, version: 1)
[2] CONCATENATION (code: 2, dep_code: 2, version: 1)
[3] MAX_POOL_2D (code: 17, dep_code: 17, version: 1)
[4] SOFTMAX (code: 25, dep_code: 25, version: 1)
[5] RESHAPE (code: 22, dep_code: 22, version: 1)
```

따라서 우리는 test를 진행할 op에 대한 list를 만들었습니다.

```
11 operators

[0] AVERAGE_POOL_2D
[1] CONV_2D
[2] DEPTHWISE_CONV_2D
[3] SOFTMAX
[4] RESHAPE
[5] ADD (code: 0, dep_code: 0, version: 1)
[6] MAX_POOL_2D (code: 17, dep_code: 17, version: 1)
[7] MUL (code: 18, dep_code: 18, version: 1)
[8] PAD (code: 34, dep_code: 34, version: 1)
[9] MEAN (code: 40, dep_code: 40, version: 1)
[10] CONCATENATION (code: 2, dep_code: 2, version: 1)
'''

각 항목에 대해 
we will start with manually writing tests, and then check if it is feasible to make a tool to automatically generate test data.

**Comment by FantBlog** [comment]
LGTM
